diab_medications/models.py

```
@@ -1,10 +1,12 @@
+# ruff: noqa: RUF002
```

Ruff (0.12.2)

Unused `noqa` directive (unused: `RUF002`)

Remove unused `noqa` directive

(RUF100)